### PR TITLE
[BUG] ROKWIRE_API_CLIENT_ID environment variable naming conflict

### LINKED ISSUE #842: [BUG] ROKWIRE_API_CLIENT_ID environment variable naming conflic
**Describe the bug**
It looks like the `ROKWIRE_API_CLIENT_ID` environment variable is being used for both the ROKWIRE_AUTH_HOST  and the ROKWIRE_ISSUER issuers, although these each use a different audience value. These need to be split into to environment variables to get this working correctly with both.
 
**To Reproduce**
1. Make request to Events BB in test environment with Core BB token authorization
2. Receive 401 unauthorized

**Expected behavior**
The expected behavior is that both the Core BB tokens and the Rokwire issuer tokens are accepted and the audience of each is validated successfully.

### PR DESCRIPTION
**Pull request by jsafoan**
## Description
`ROKWIRE_API_CLIENT_ID` is split into to two environment variables for the audience of ROKWIRE_ISSUER and ROKWIRE_AUTH_HOST tokens.

**Fixes #842 **

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My change requires updating the documentation.
- [x] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Hi, @ywkim312 and @minump,

This PR adds a new env variable for the ROKWIRE_AUTH_HOST.  I'm reviewing this PR and its integration with all Building Blocks that we maintain. If there are no major concerns, we need to soon make another release and update all the Building Blocks with this new environment variable. Thanks.

**Comment by ywkim312** [comment]
I am testing with core token and keep having this `2021-12-08T15:49:04.463Z WARNING [Thread-2  ] : auth_middleware - jwt error on decode. message = Signature has expired` even though I am using the freshly created ones.

**Comment by ywkim312** [comment]
Maybe, I only need to use API key for profiles, not core token....

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Contributions Catalog login callback endpoint to /catalog/auth/callback. [#803](https://github.com/rokwire/rokwire-building-blocks-api/issues/803) 
 
 ### Fixed
+- Added additional env variable for audience for ROKWIRE_AUTH_HOST tokens. [#842](https://github.com/rokwire/rokwire-building-blocks-api/issues/842)
```
Comment (sandeep-ps) [inline CHANGELOG.md:12]: One minor suggestion. To keep the chronological order of changes in the CHANGELOG, could you please move this entry to the bottom of the Fixed section? Thanks.

**Comment by ywkim312** [comment]
It worked after completely removing my virtualenv and recreated. Maybe somethings got messed up in there.

**Comment by ywkim312** [comment]
Both logging and profiles worked okay. Contribution doesn't have anything with CORE token. @sandeep-ps Please confirm there is no core token related in contribution building block

File: CHANGELOG.md (hunk 1 above)
Comment (jsafoan) [reply CHANGELOG.md]: Sure. Thanks for the suggestion.

**Comment by ywkim312** [comment]
Events building block worked okay as well.

**Review by ywkim312 — APPROVED** [review]

**Comment by sandeep-ps** [comment]
> Both logging and profiles worked okay. Contribution doesn't have anything with CORE token. @sandeep-ps Please confirm there is no core token related in contribution building block

Yes, that's right.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**